Bucketeer2/Sources/Internal/Model/Evaluation.swift

```diff
@@ -10,3 +10,35 @@ struct Evaluation: Hashable, Codable {
     let reason: Reason
     var variation_value: String
 }
+
+extension Evaluation {
+    func getVariationValue<T>(defaultValue: T, logger: Logger?) -> T {
+        let value = self.variation_value
+        let anyValue: Any?
+        switch defaultValue {
+        case is String:
+            anyValue = value
+        case is Int:
+            anyValue = Int(value)
+        case is Int64:
+            anyValue = Int64(value)
+        case is Float:
```

Comment:
I just noticed we don't need the Float and Int64 cases.

https://github.com/ca-dp/bucketeer-ios-sdk/pull/87/files#diff-ecc95802cf8595971f48fdbb7da52ef1181e037b9dd20132058a84345117f4e4R108-R126

Reply:
You are right! Deleted.
https://github.com/ca-dp/bucketeer-ios-sdk/pull/87/commits/bcdf73d0d9a296c2abc5a685d086c1cd319997a0

Reply:
Thank you!